server/src/repositories/teams.repository.ts

```diff
@@ -83,4 +84,36 @@ export class TeamRepository extends Repository<Team> {
 	deleteOneById(id: string) {
 		return this.delete(id);
 	}
+
+	// eslint-disable-next-line consistent-return
+	async addPeopleToTeam(id: string, users: UserProfile[]) {
+		try {
+			const team = await this.createQueryBuilder('Team')
+				.where('Team.id = :id', { id })
+				.leftJoinAndSelect('Team.users', 'Users')
+				.getOne();
+
+			if (!team) {
+				throw new Error('Team was not found');
+			}
+			// eslint-disable-next-line array-callback-return
+			users.map((user: UserProfile): void => {
+				const isExist = team.users?.find((el) => el.id === user.id);
+				if (!isExist) {
+					team?.users?.push(user);
+				}
+			});
+
+			await this.save(team);
```

Comment:
I suppose, this should return updated team. You don't need to do select query again

Reply:
yeah. you're rigth